modules/oc-mirror-installing-plugin.adoc

```diff
@@ -3,6 +3,15 @@
 // * installing/disconnected_install/installing-mirroring-disconnected.adoc
 // * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
 
+ifeval::["{context}" == "installing-mirroring-disconnected"]
+:oc-mirror:
+endif::[]
+
+ifeval::["{context}" == "about-installing-oc-mirror-v2"]
+:oc-mirror-v2:
+:restricted:
```

Comment:
You've defined 'restricted' here but haven't used it anywhere. You also haven't undefined it at the end of the file. I think it should be removed from here

Reply:
Ack.